src/plugins/content_management/server/event_stream/tests/util.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const tick = (ms: number = 1) => new Promise((r) => setTimeout(r, ms));
+
+export const until = async (check: () => boolean | Promise<boolean>, pollInterval: number = 1) => {
```

Comment:
Shouldn't we have a time out if the check never succeed?

Reply:
Nope, this is used in testing. If check never succeeds, the test fails, usually after 5 sec, or whatever custom is configured.